kratos/mpi/tests/test_distributed_sparse_matrices.py

```diff
@@ -63,9 +67,6 @@ class TestDistributedSparseMatrices(KratosUnittest.TestCase):
         (17,10):1.0,(17,36):1.0,(17,7):1.0,(10,17):1.0,(10,36):1.0,(10,7):1.0,(36,17):1.0,(36,10):1.0,(7,17):1.0,(7,10):1.0,(25,30):1.0,(25,9):1.0,(14,30):1.0,(30,25):1.0,(30,14):1.0,(30,9):1.0,(9,25):1.0,(9,30):1.0
     }
 
-    bref_all = np.array([1,3,2,3,3,3,5,6,4,4,2,4,3,1,6,3,5,3,6,3,0,1,4,1,4,4,6,1,5,2,2,3,1,4,1,5,2,4,1,3])

```

Comment:
why did u take out this one?

Reply:
its not being used anywhere and it gives problems as it is not inside the skipped test and it uses numpy.